src/shoobx/mocks3/models.py

```diff
@@ -182,6 +179,15 @@ def etag(self):
                 self._etag = file_hash.hexdigest()
         return f'"{self._etag}"'
 
+    @property
+    def checksum_value(self):
+        return self._etag
+
+    @checksum_value.setter
+    def checksum_value(self, value):
+        self.etag
```

Comment:
huh, maybe self.etag=value and not pass?

Reply:
why use the value? `self.etag` already creates an md5 hash.

Reply:
done